Fix #216 transpile object destructuring

Freakazo opened the pull request:
Fix for issue: #216

Added the babel transform-es2015-destructuring plugin and configured it in package.json.

It didn't seem like package-lock.json needed to be updated.

haveyaseen commented:
Just one small thing: Any particular reason you added it to `.babelrc`, too?  It is already depended on by `babel-preset-env` which is why `package-lock.json` did not need to be updated.

Freakazo commented:
@haveyaseen I added it to the babelrc because it looked in sync with the command line arguments for the transpile:es script.

Honestly I'm not sure how or when the .babelrc file is used.

Removed the changed to the .babelrc in the above commit.
